### ISSUE TITLE
Agrarian hoburg pod farmers

### ISSUE THREAD

**Issue by Ealb** [elmokki/nationgen#105]
Not for 0.6.0 since it's too involved, but:
- mostly agrarian/sometimes primitive hoburg theme(s)
- cultivated vinemen troops
- symbiotic vineman/ogre hoburg "bio-exoskeletons"
- no secondary race mages
- secondary race filters that turn some into reduced-stat & cost #slave "grafted" slaves, some into semi-reduced/semi-increased stat and/or cost #mindless #magicbeing "germinated" thralls
- possibly minor popkill of a level that growth can offset - this requires testing, and might be best as a probabilistic national filter so not all of them do it (that applies to various combinations of these elements as well - not necessarily all should grow brainwashing parasite plants, *and* vinemen, *and* organic mecha) 
- it also might be a better idea to split this into two mostly-agrarian themes - on the one hand symbiotic cultivated vinemen allies and "vine armor", and on the other evil parasitic farm-grown slaver-plant hoburg puppetmasters)
- basically take Zonk's original offhand comment that inspired the clockwork armor ("would vinemen with gear be too out there?" or words to that effect) and bring it full circle, while giving agrarian-techlevel hoburgs their own special brand(s) of evil since they don't get clockwork stuff

(defined as an issue to write out the basic outline so I can remember it w/o impulsively throwing aside everything and working on them right now)

**Comment by elmokki** [comment]
This sounds like an excellent idea to make the agrarian hoburgs more attractive. Hoburgs sure are getting a lot of love.

Just by the way, I think the theme system should eventually used for two far less interesting things: Greek style (bronze) humans and Caelian subraces. I think good enough Caelian subraces can actually happen by just making a separate basesprite for each for troop poses and using themes and chanceincs to influence different mix probabilities. For mages either the same with some more complex chanceincs or something. Ideal would obviously be proper subrace handling, but while coding the things that effectively work as having different basesprites is easy, figuring out how to implement that for generation is a bit different.

**Comment by elmokki** [comment]
Neither of those is a priority of any kind. If they were, I'd make them. There's hell of a lot of stuff for 0.6.0 already.

**Comment by Ealb** [comment]
Another good side of doing hoburg pod people is that the data definitions for their secondary races would heavily overlap with what would be needed for mindless illithid slaves.

I'd agree that properly Greek bronze-ages humans are something that should happen eventually. When I do the spritework for the "central" hoburgs I may have some work that can bleed over in that direction, even if only a few more hoplons and possibly bits for human automata derived from the hoburg's bronze clockwork junk.

(And yes, obviously, Caelians too.)

I have more work done than is uploaded, but I don't have a good, well-rounded set done so I've held off from committing it. This week turned out to give me less free time than expected (especially with the American holiday over the weekend), but next week *should* be better.

**Comment by Ealb** [comment]
Preliminary proof-of-concept/design art/prototype/what-have-you for parasitic and symbiotic vine armors:

![basehoburg_vine_parasite](https://cloud.githubusercontent.com/assets/12613576/8773046/34ed2234-2e9f-11e5-883e-a8d059f7e6ff.png)          ![basehoburg_vine_symbiote](https://cloud.githubusercontent.com/assets/12613576/8773049/374186f6-2e9f-11e5-892c-f4f03f06689b.png)

**Comment by elmokki** [comment]
Those do look great.

I do think that maybe they should be a bit less numerous in rosters than the clockwork things for advanced hoburgs. That said, I bet most people recruit like 1 or at most 2 type of clockworks  only so having less of these in roster is more a visual than game balance thing, except of course each nation individually having less change of getting the best possible configuration. I don't, however, think they should be elite/sacred only by any means.

**Comment by Ealb** [comment]
At the most, I'm going to make two sizes of these, and I'll probably put a max unit and/or variety limit in place like the clockworks used to have.

I'd agree that most people would only use one or maybe two poses - that's been how I played them SP, and it also fits with my limited NG MP experience even outside of the clockworks. Dominions can rather strongly encourage players to ignore most of a nation's roster IOT focus on several units, and I've seen that to be even moreso the case in NG. - it makes sense, since with NG you may well have outright picked the nation on the basis of having this or that unit (well, or mage). The last game I finished, which ended with a Domkill around turn 93 I think, probably ~60% of all units I recruited were my nations' medium infantry (recruit-anywhere DW hammer lizards with berserk +2 and full ring; I wanna say they were 10g and 12r - a steal, even with completely average lizard stats), plus ~15% assorted indies (mostly xbows and ichtyids, with a side of pale ones and lavaborn), ~10% for elite cap-only light infantry, ~10% for ringmail composite bow/broadsword rangers, 5% for assorted national siege chaff/chariots/armored Xbows, and 0% for sacreds. Honestly, it might have been 80% that were hammerers.

**Comment by elmokki** [comment]
Yes, the whole NationGen troop roster philosophy has been to generate a roster where you can pick the couple of most useful units. Just like vanilla. I don't see it as a problem at all as I said - it's more of a visual thing. The troop templates are there to occasionally mix it up, but they may also make rosters more one sided occasionally as well.
